bin/i18n/sync-out.rb

```diff
@@ -258,6 +258,21 @@ def sanitize_data_and_write(data, dest_path)
   File.write(dest_path, dest_data)
 end
 
+def sort_and_sanitize(hash)
```

Comment:
This is a pretty useful function, I would recommend moving it to `I18nScriptUtils`